MANUAL.txt

```diff
@@ -3561,6 +3561,19 @@ literally.  In writing LaTeX, enabling `smart` tells pandoc
 to use the ligatures when possible; if `smart` is disabled
 pandoc will use unicode quotation mark and dash characters.
 
+### Extension: `smart_quotes` ###
+
+Interpret straight quotes as curly quotes.
+
+FIXME: explain
+
+### Extension: `special_strings` ###
+
+Interpret `---` as em-dashes, `--` as en-dashes, and `...` as
+ellipses.
+
+FIXME: explain
+
```

Comment:
FIXME should be fixed.  Is there also a change that should be made to `smart`, to indicate that it is a synonym for `+smart_quotes+special_strings`?